### DISCUSSION TITLE
addons+support@mozilla-community.org

### DISCUSSION THREAD

**Discussion by NitroFuelCoupe** [Authenticator-Extension/Authenticator#630] [General]
Hi, I have been using this app for years and I love it. 
Recently I had software problems and lost this app, authenticator, so I added a new one from Firefox as I use it at Facebook. 
No matter what I try I just can't get it to work.

I can get to Facebook, add my password, and then the authenticator opens and I click on, click the pencil, click the + mark and add Facebook as my issuer but can't get it to give me my secret and have been trying for days now. 

Any help you could please give me would be greatly appreciated. 

I recently suffered a near fatal accident and not everything 
comes easy to me yet.

Thank you.

Robert Welch
CEO
Global Technology Inc.
gtmci@yahoo.com

Thank you once again.

**Comment by mymindstorm** [comment]
> Recently I had software problems and lost this app, authenticator,

To clarify, do you mean that your data was lost and you can't login to Facebook now? 

> I can get to Facebook, add my password, and then the authenticator opens and I click on, click the pencil, click the + mark and add Facebook as my issuer but can't get it to give me my secret and have been trying for days now.

To add the account you will need to go to your Facebook security settings and change the setting "Use two-factor authentication"

https://www.facebook.com/settings?tab=security

**Reply by NitroFuelCoupe** [reply]
**_How do I go to Facebook Security Settings when I cannot log in without the 2-factor authenticator I had before? 
FB S.S. is located inside the FB acct. and without 2 factor it is not possible to log in now. 
All because FF changed the way the add ons worked before and then suddenly all of my add ons no longer worked. 
Now I can no longer get into my 7-year-old FB account._**

**Reply by mymindstorm** [reply]
https://authenticator.cc/docs/en/lost-codes
